dashboard/app/config.go

```diff
@@ -26,8 +26,10 @@ import (
 type GlobalConfig struct {
 	// Min access levels specified hierarchically throughout the config.
 	AccessLevel AccessLevel
-	// Email suffix of authorized users (e.g. "@foobar.com").
+	// Deprecated - Email suffix of authorized users (e.g. "@foobar.com").
```

Comment:
FYI @dvyukov

Reply:
I was thinking of leaving it for some more time so that downstream users get a chance to migrate too but if we are ok with more disruptive upgrades (I guess the config will just not compile anyway) we could do it early yes. Or maybe we don't even need a migration phase ?

Reply:
I think we can remove it right away (if we combine it with updating the syzbot config), but I don't have a strong preference.